WRN-12700: Cross-Platform: Fixed Input to show keypad properly when screen width is narrow (#1135)
enactjs/sandstone: CHANGELOG.md

- `sandstone/BodyText` font-size for size `small` and RTL locale
- `sandstone/Input.InputField` size 'small' line-height to center text vertically

> - `sandstone/Input.Keypad` keypad margins to show all digits and body padding to show title for smaller screens

Reviewer: ```suggestion
- `sandstone/Input` to show title and keypad properly when `type` is `number` and screen width is narrow
```
